### ISSUE TITLE
Add tiny explosions when expeditions land

### ISSUE THREAD

**Issue by wschella** [ZeusWPI/MOZAIC#78]

**Comment by tl3ilaxu** [comment]
Was thinking of letting the expedition join the orbit around the planet. Wouldn't that make more sense since the fleets really also leave from the orbit. Maybe explode the enemy fleet that is in orbit.

**Comment by wschella** [comment]
Yes, but that is less easy! Exploding enemy fleets can only be done on takeover.

**Comment by tl3ilaxu** [comment]
Yeah ok let them explode in orbit then depending on successfull takeover

**Comment by wschella** [comment]
Would be nice. But that deserves it's own issue I think.
